### DISCUSSION TITLE
Preconfigured Deployment on Client

### DISCUSSION THREAD

**Discussion by trashmanbrah** [rustdesk/rustdesk#1351] [General]
It would be a nice because you still need to name the .exe file to the host and key. I still have to have put in the ID on the client.

**Comment by DarkZoneSD** [comment]
You could do something a bit messy like writing a PS script that Invokes a Webclient and downloads RustDesk from its website to a Destination Path, then rename the .exe with your ID and Key and start said exe. Then convert the whole thing to an exe via PS2EXE or a similar tool and there you finally have everything combined into one exe file.

**Comment by gointern** [comment]
I would prefer if server info would be built into rustdesk exe. Modifying exe resource to add that information. Filename is clean and can be renamed, no need for scripts, easy to manage.
